### PR TITLE
Added Re-issuing internal certificates topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on content from https://help.stackrox.com/docs/configure-stackrox/reissue-internal-certificates/

Preview: https://openshift-docs-git-reissue-internal-certificates-gnelson.vercel.app/openshift-acs/master/configuration/reissue-internal-certificates.html

Following modules used in this PR are part of https://github.com/openshift/openshift-docs/pull/33062
- modules/restart-scanner-container.adoc
- modules/restart-central-container.adoc

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer PTAL

**Comment by bergerhoffer** [comment]
There is also a conflict in the PR that you'll need to fetch/rebase/resolve.

File: configuration/reissue-internal-certificates.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+These certificates have expiration dates, and you must re-issue them before they expire.
+You can view the certificate expiry dates in the *Platform Configuration* -> *Clusters* view.
+
+//Add link to role based permissions and resources
+
+//Re-issue internal certificates for Central
+include::modules/reissue-internal-certificates-central.adoc[leveloffset=+1]
+
+//Restart Central
+include::modules/restart-central-container.adoc[leveloffset=+2]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply configuration/reissue-internal-certificates.adoc:19]: Yup, they are part of another PR https://github.com/openshift/openshift-docs/pull/33062
- https://github.com/gaurav-nelson/openshift-docs/blob/add-cert-auth/modules/restart-central-container.adoc
- https://github.com/gaurav-nelson/openshift-docs/blob/add-cert-auth/modules/restart-scanner-container.adoc

**Comment by gaurav-nelson** [comment]
Thanks @bergerhoffer 

The missing modules are part of another PR #33062
- https://github.com/gaurav-nelson/openshift-docs/blob/add-cert-auth/modules/restart-central-container.adoc
- https://github.com/gaurav-nelson/openshift-docs/blob/add-cert-auth/modules/restart-scanner-container.adoc

Should we merge that one first?

**Comment by bergerhoffer** [comment]
> Should we merge that one first?

Yep, we should. I'll do a review of that one now.

**Comment by bergerhoffer** [comment]
I re-reviewed and everything else looks good in this PR now. So once the other PR is merged (and this one rebased), should be good to go!

**Comment by bergerhoffer** [comment]
Thanks, I see the modules from the other PR. LGTM, merging

**Pull request merged**